applications/migrations/0001_initial.py

```diff
@@ -45,7 +45,7 @@ class Migration(migrations.Migration):
                 ('devpost', models.URLField(blank=True, null=True)),
                 ('linkedin', models.URLField(blank=True, null=True)),
                 ('site', models.URLField(blank=True, null=True)),
-                ('diet', models.CharField(choices=[('None', 'No requirements'), ('Vegeterian', 'Vegeterian'), ('Vegan', 'Vegan'), ('No pork', 'No pork'), ('Gluten-free', 'Gluten-free'), ('Others', 'Others')], default='None', max_length=300)),
+                ('diet', models.CharField(choices=[('None', 'No requirements'), ('Vegetarian', 'Vegetarian'), ('Vegan', 'Vegan'), ('Gluten-free', 'Gluten-free'), ('Others', 'Others')], default='None', max_length=300)),
```

Comment:
Don't delete older schemas, please create a new migration to remove the old option; this way already created dbs modify their structure, otherwise believes this change was already done.

Reply:
same applies for all migration files modified